content/en/docs/what-is-grpc/faq.md

```diff
@@ -31,9 +31,7 @@ Google has been using a lot of the underlying technologies and concepts in gRPC
 
 ### Which programming languages are supported?
 
-See [Officially supported languages and platforms][].
-
-[Officially supported languages and platforms]: /about/#officially-supported-languages-and-platforms
+See [Officially supported languages and platforms](https://grpc.io/docs/languages/).
```

Comment:
Looks like https://grpc.io/docs/#official-support is the 1:1 replacement for this link, as the table moved. Both links have the same list of languages, but the table has details about the platforms which is part of the link text. But the table doesn't have links to the documentation per-language.

I think either one is better than what we have. Would you mind either changing to link text to better match the url "See \[Supported languages\](...)" or leave the text the same and change the url to https://grpc.io/docs/#official-support ?